src/System.Private.Reflection.Core/src/System/Reflection/Runtime/MethodInfos/RuntimeMethodInfo.cs

```diff
@@ -173,7 +173,7 @@ public sealed override object Invoke(object obj, BindingFlags invokeAttr, Binder
 
         public abstract override MethodInfo MakeGenericMethod(params Type[] typeArguments);
 
-        public sealed override int MetadataToken
+        public override int MetadataToken
```

Comment:
Similar comment regarding this as with the RuntimeConstructorInfo - I really like to minimize these non-sealed overrides.